### PR TITLE
chore: make style sheets consistent

### PR DESCRIPTION
**Pull request by liyasthomas**
Closes HFE-63

TLDR;
1. Adds missing PWA icons
2. Adds `supported-color-schemes` meta tags for dark/light mode detection
3. Adds `color-scheme` in the themes' style sheet for native dark/light mode form inputs
4. Made style sheets consistent across the `hoppscotch-ui` and `hoppscotch-common` packages

### Summary
🎨🌐🛠️

This pull request improves the web app's appearance, performance, accessibility, and integration with different color schemes, browsers, and operating systems. It adds the `color-scheme` property to the `themes.scss` files of different packages, updates the background color and the theme-color meta tags, modifies the title and the icons of the web app, and adds custom protocol handlers to the web app manifest. It also enhances the UI elements of the code editor, the select element, and the split panes, and refactors some code related to the font size, line height, and sticky fold dimensions.

> _`color-scheme` added_
> _web app looks better, faster_
> _autumn leaves falling_

### Walkthrough
*  Add `color-scheme` property to different theme selectors to optimize the user interface for light and dark themes ([link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-c80409917a5fc9fa688ecbb372869961d5b7b33bf6f53ddacacfdf84df85fe8eR210), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-c80409917a5fc9fa688ecbb372869961d5b7b33bf6f53ddacacfdf84df85fe8eR216), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-c80409917a5fc9fa688ecbb372869961d5b7b33bf6f53ddacacfdf84df85fe8eR222), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-8a6451b28ac835ec8728b525a3185a4f9104072818652840044ae4c839db7d82R210), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-8a6451b28ac835ec8728b525a3185a4f9104072818652840044ae4c839db7d82R216), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-8a6451b28ac835ec8728b525a3185a4f9104072818652840044ae4c839db7d82R222), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-01add05769746b59667d8fcba29a674ddbbdc6b669473474ef8e0858bd996bb0R210), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-01add05769746b59667d8fcba29a674ddbbdc6b669473474ef8e0858bd996bb0R216), [link](https://github.com/hoppscotch/hoppscotch/pull/3074/files?diff=unified&w=0#diff-01add05769746b59667d8fcba29a674ddbbdc6b669473474ef8e0858bd996bb0R222)) in `themes.scss` files

### CODE REVIEW

**Review by JoelJacobStephen — APPROVED** [review]
LGTM!!

**Review by anwarulislam — APPROVED** [review]
LGTM 🚀
